app/views/users/delete_success.erb

```diff
@@ -0,0 +1,11 @@
+<!doctype html>
```

Comment:
The layout.erb file takes care of the HTML boilerplate for you. So you should not include anything in your view files that is already in the layout file.